### PR TITLE
feat: adding a toggle switch for left panel items

### PR DESCRIPTION
**Pull request by anu1097**
New UI -
Screenshot 2021-02-26 at 3 17 54 AM

Description - Saw a good first issue - https://github.com/AmruthPillai/Reactive-Resume/issues/323

Approach taken - Read through the code to understand the data flow. Not the cleanest solution, but does the job.
Thought about creating a generic function or modify safety-check function to avoid code duplication. 
But that would require significant changes hence went against it and used this approach.

### CODE REVIEW

**Comment by anu1097** [comment]
@AmruthPillai can you have a look. I think its ready to be merged.

File: src/contexts/ResumeContext.js [hunk 1]
```diff
@@ -68,6 +68,18 @@ const ResumeProvider = ({ children }) => {
           debouncedUpdateResume(newState);
           return newState;
 
+        case 'on_toggle_use_item':
+          items = get(state, payload.path);
+          index = findIndex(items, ['id', payload.value.id]);
+          if ('isVisible' in items[index]) {
+            items[index].isVisible = !items[index].isVisible;
+          } else {
+            items[index].isVisible = false;
```
Comment (amruthpillai) [inline src/contexts/ResumeContext.js:77]: If `isVisible` doesn't exist in the data array, we should have all sections visible by default. Here, when a user tries to import/load an older version of the resume, they wouldn't see any of the sections.

**Review by AmruthPillai — CHANGES_REQUESTED** [review]
This needs to be addressed before we can merge :)

File: src/contexts/ResumeContext.js (hunk 1 above)
Comment (anu1097) [reply src/contexts/ResumeContext.js:77]:
```
export const isItemVisible = (section) =>
  section && section.isVisible !== false;
```

I am using this logic to determine whether to display or not. if IsVisible is not present in existing section. This function will return true. So this should handle it.

**Pull request merged**